.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version
+      - image: circleci/golang:1.14
```

Comment:
@claygoddard Do you think we need to add any more versions? I was thinking that since we release a binary, testing multiple versions isn't that important.

Reply:
This is fine with me!